scripts/quests/ahtUrhgan/Led_Astray.lua

```diff
@@ -0,0 +1,189 @@
+-----------------------------------
+-- Led Astry
+-- Log: 6, Quest: 65
+-- Mhasbaf:         !pos 54.701 -6.999 11.387 50
+-- Whitegate Region !pos 71 0 9.299 50
+-- Tohka Telposkha: !pos 22.1 0 22.759 50
+-- Rubahah:         !pos -96.746 0 -14.701 50
+-- Cacaroon:        !pos -72.026 0.000 -82.337 50
+-- Whitegate Region !pos 12.5 -94 2 50
+-- Tataroon:        !pos -25.189 0 -39.022 53
+-----------------------------------
+require('scripts/globals/items')
+require('scripts/globals/quests')
+require('scripts/globals/npc_util')
+require('scripts/globals/interaction/quest')
+require('scripts/globals/titles')
+require('scripts/globals/zone')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.AHT_URHGAN, xi.quest.id.ahtUrhgan.LED_ASTRAY)
+
+quest.reward =
+{
+    item = xi.items.IMPERIAL_SILVER_PIECE,
+}
+
+quest.sections =
+{
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE
+        end,
+
+        [xi.zone.AHT_URHGAN_WHITEGATE] =
+        {
+            ['Mhasbaf'] = quest:progressEvent(808),
+
+            onEventFinish =
+            {
+                [808] = function(player, csid, option, npc)
+                    quest:begin(player)
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED and vars.Prog == 0
+        end,
+
+        [xi.zone.AHT_URHGAN_WHITEGATE] =
+        {
+            ['Mhasbaf'] = quest:event(834),
+
+            onRegionEnter =
+            {
+                [7] = function(player, region)
+                    return quest:progressEvent(809)
+                end,
+            },
+
+            onEventFinish =
+            {
+                [809] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Prog', 1)
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED and vars.Prog == 1
+        end,
+
+        [xi.zone.AHT_URHGAN_WHITEGATE] =
+        {
+            ['Mhasbaf'] = quest:event(834),
+
+            ['Tohka_Telposkha'] = quest:progressEvent(810),
+
+            onEventFinish =
+            {
+                [810] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Prog', 2)
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED and vars.Prog == 2
```

Comment:
Doesn't appear that we can escape a Prog value of 2, as nothing pushes this to 3 within the block

Reply:
I must have missed a number in my numbering refactoring. I'll re verify all steps tonight

Reply:
ran through and found I had removed 2 needed blocks of code checking for varBits